backend/src/routes/api/status/statusUtils.ts

```diff
@@ -56,39 +64,12 @@ export const status = async (
         clusterID,
         clusterBranding,
         isAdmin,
+        isAllowed,
```

Comment:
The more I think of it the more I would like to see this named something more meaningful. `isAllowed` is allowed to do what?

Clearly they can look at the resources without this flag, so it's not `isAllowed` to use the Dashboard.

What is our goal with this flag? Is allowed to do meaningful actions -- like creating notebooks and whatever else we add in the future (baring anything that we want to grant to just admins).

Maybe `isAllowedOwnership`? `isAuthenticated` (maybe too broad)? `canCreateResources`?

Reply:
We can go with `isAllowed` for now -- it's unique enough to replace with a better name when we can think of it.